docs/proposals/high-availability.md

```diff
@@ -0,0 +1,34 @@
+# High Availability of Daemons in Kubernetes
```

Comment:
Please be more specific about which daemons-- arguably apiserver should have a different strategy than scheduler and controller-manager.